apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -202,8 +202,10 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
   };
 
   return (
-    // TODO: Figure out why making this view accessible breaks element querying on iOS.
-    <RootView style={themedStyles.root} accessible={Platform.OS !== 'ios'} testID={ROOT_VIEW}>
+    // TODO: Setting the testID should disable React Native's layout-only view removal optimization.
+    // However, for win32, the view removal still happens even though the testID is being set on this view.
+    // As a temporary patch, set the view to be accessible on win32 to ensure that the view is not removed.
+    <RootView style={themedStyles.root} accessible={Platform.OS === ('win32' as any)} testID={ROOT_VIEW}>
```

Comment:
I believe that whole "layout-only view removal" is actually controlled by the prop `collapsable`, at least In React Native iOS/Android. Maybe we can just set that to be true? @lawrencewin FYI

Reply:
There was a conference talk / blog about it somewhere. I can try and dig it up if it helps.

Reply:
https://reactnative.dev/architecture/view-flattening
https://reactnative.dev/docs/view#collapsable-android

This was an optimization initially made in Fabric for Android, that they then brought to iOS. My guess is this is what the win32 code is referring to, and that the website docs are slightly out of date (I don't think it's Android only).

Reply:
From a very brief investigation into "layout-only view removal"
- couldn't find anything in the react native docs
- found this on stack overflow, take with a grain of salt

> When RN creates the native views from the shadow nodes tree - it performs some optimizations. Views that do not actually show on screen (don't draw anything or just used in JSX to contain and layout their children) can be removed when constructing the native hierarchy. This is why they're called "layout-only" views.

- there are other IDs that apparently also disable the 'layout-only view removal' optimization for this view - [id](https://reactnative.dev/docs/next/view#id) and [nativeid](https://reactnative.dev/docs/next/view#nativeid)

Reply:
@Saadnajmi sorry I sent my comment before I saw your second and third ones. Thanks for finding that! I can try setting collapsable to be false on this view